autoscheduler/frontend/src/.eslintrc.js

```diff
@@ -45,5 +45,6 @@ module.exports = {
   rules: {
     "react/jsx-filename-extension": [1, {"extensions": [".jsx", ".tsx"]}],
     "import/no-extraneous-dependencies": ["error", {"devDependencies": ["**/*.test.ts", "**/*.test.tsx"]}],
+    "react/prop-types": 0
```

Comment:
What does this rule specify? Is this related to #53 ?

Reply:
Yes.